docs/sql-migration-guide.md

```diff
@@ -52,6 +52,8 @@ license: |
 
   - In Spark 3.1, refreshing a table will trigger an uncache operation for all other caches that reference the table, even if the table itself is not cached. In Spark 3.0 the operation will only be triggered if the table itself is cached.
 
+  - In Spark 3.1, we support typed literal as partition column value, such as `PARTITION (dt = date '2020-01-01')`, it will be treated as partition column value `2020-01-01`. In Spark 3.0 the partition value will be treated as `date '2020-01-01'`.
```

Comment:
> we support typed literal as partition column value

Do you mean `we support a typed literal for binary, date, timestamp, and interval in the INSERT INTO statement`?

> it will be treated as partition column value `2020-01-01`. In Spark 3.0 the partition value will be treated as `date '2020-01-01'`.

What does the statement above mean? For instance, the partition value of the example query you added in the doc page is null;
```
scala> sql("""CREATE TABLE students (name STRING, address  STRING) PARTITIONED BY (birthday DATE);""")
scala> sql("""INSERT INTO students PARTITION (birthday = date'2019-01-02') VALUES ('Amy Smith', '123 Park Ave, San Jose');""")
scala> sql("""SELECT * FROM students;""").show()
+---------+--------------------+--------+
|     name|             address|birthday|
+---------+--------------------+--------+
|Amy Smith|123 Park Ave, San...|    null|
+---------+--------------------+--------+
```
Could you make it clearer so that users can understand smoothly?

Reply:
Updated, now maybe more clear?